{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":": Timeout - Async callback was not invoked within the 40000 ms timeout specified by jest.setTimeout.","children":[{"name":"update file","uid":"6c2d04805b702e19","parentUid":"4eb82bc3d9494a40d5aa611d3cd64d44","status":"failed","time":{"start":1707297469924,"stop":1707297509927,"duration":40003},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"4eb82bc3d9494a40d5aa611d3cd64d44"},{"name":"Error: Caught error after test environment was torn down\n\nTimeout (10000) for task exceeded","children":[{"name":"signup should create user","uid":"b57ebbb327ffa53","parentUid":"7ca522a4d2afd44f0c649c607d16d20d","status":"failed","time":{"start":1707297514249,"stop":1707297515196,"duration":947},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"7ca522a4d2afd44f0c649c607d16d20d"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}